src/user-guide/writing-workflows/scheduling.rst

```diff
@@ -2219,15 +2219,12 @@ executing, e.g:
 Runahead Limiting
 -----------------
 
-Runahead limiting prevents a workflow from getting too far ahead of the oldest
-active cycle point by holding back tasks in cycles beyond a specified limit.
+Runahead limiting prevents workflow activity from getting too far ahead of the
+earliest :term:`active cycle`, by holding back tasks in cycles beyond a
+configurable limit.
 
-The runahead limit is defined as an interval measured from the oldest active cycle.
-A cycle is considered to be "active" if it contains any :term:`active` tasks
-(e.g. running tasks).
-
-Tasks in cycles which are beyond the limit are called :term:`runahead` tasks
-and are displayed in the GUI/Tui with small circle above them:
+Tasks in cycles beyond the runahead limit are called "runahead limited" and are
+displayed in the GUI/Tui with small circle above them:
```

Comment:
Sadly this is not true, see https://github.com/cylc/cylc-flow/issues/5580

Reply:
Ah, yeah, I did know that, I was just tweaking the existing text on autopilot there. I'll rephrase it.

Reply:
Now this:
```
Runahead limiting confines workflow activity to a configurable number of cycles
beyond the earliest :term:`active cycle`.

Tasks in the :term:`n=0 window <n-window>` at the runahead limit are actively
held back, and are displayed in the GUI/Tui with a small circle above them.

.. note::

   Tasks in the :term:`n=1 window <n-window>` are not displayed as runahead
   limited; they just form the future graph and are not yet being actively limited.
```